### ISSUE TITLE
Changing faces in customize has no effect

### ISSUE THREAD

**Issue by hexmode** [magit/magit-popup#14]
My default face is set to a proportional font.  I try to set those faces that depend on a fixed-pitch font to fixed-pitch.  Customizing the faces in the `magit-popup-faces` group to use a fixed-pitch font (in this case, Go Mono) seems to have no effect even after saving my customizations and restarting emacs.

**Comment by tarsius** [comment]
Have you updated Magit recently? Magit uses `transient` instead of `magit-popup` now, so you have to customize the former to affect Magit's transients (formerly known as popups).

**Comment by hexmode** [comment]
Define "recent".  This was with the version from MELPA tagged 20190307.  Updating now to ...09

**Comment by hexmode** [comment]
I have this installed still becuase it is required by tblui, magit-gitflow, docker.el, and, apparently, maggit-gerrit.

I say apparently because I invoked magit-gerrit and saw the faces I had selected.

**Comment by tarsius** [comment]
> I say apparently because I invoked magit-gerrit and saw the faces I had selected.

So configuring `magit-popup`'s faces to "use fixed-pitch", has the desired effect on popups created with that package. Have you tried doing the same for `transient` to affect the popups/transients created with that package? I am asking because the "I say apparently" at the beginning of you sentence confuses me.

**Comment by hexmode** [comment]
Sorry for the confusion.  Until I realized that the faces were those in
the transient package, I was confused because changing them didn't
change what I saw in magit.  My use of magit-gerrit is more intermittent
than magit core, but when I saw that magit-gerrit depended on
magit-popup, I decided to check it out.

It was then that I saw the faces I had selected.  I didn't worry to much
about it and closed the issue.
